plugins/janus_videoroom.c

```diff
@@ -5501,6 +5504,97 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, janus_plugin_rtp
 	janus_videoroom_publisher_dereference_nodebug(participant);
 }
 
+/* Retrieve the current position in the ramp up ramp down process
+ * Ramp positions:
+ * Spatial 0 (L) - Temporal 15fps -> 0
```

Comment:
i think you'll run into issues with this as only chrome restricts itself to two temporal layers while other browsers will use three (which remains the default of libwebrtc)